rest/importtest/collections_import_test.go

```diff
@@ -410,6 +429,100 @@ func TestMultiCollectionImportRemoveCollection(t *testing.T) {
+	require.Equal(t, uint64(0), importVersion)
+
+	// Update with PUT, change collection set, version should change
+	response = rt.SendAdminRequest(http.MethodPut, "/{{.db}}/_config", fmt.Sprintf(collectionsDbConfig, testBucket.GetName(), string(twoCollectionConfigJSON)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(1), importVersion)
+
+	// Update with PUT, do not change collection set, version should not change
+	response = rt.SendAdminRequest(http.MethodPut, "/{{.db}}/_config", fmt.Sprintf(collectionsDbConfigRevsLimit, testBucket.GetName(), string(twoCollectionConfigJSON)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(1), importVersion)
+
+	// Upsert with POST, do not include or change collection set, version should not change
+	response = rt.SendAdminRequest(http.MethodPost, "/{{.db}}/_config", collectionsDbConfigUpsertRevsLimit)
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(1), importVersion)
+
+	// Upsert with POST, include scopes but do not change collection set, version should not change
+	response = rt.SendAdminRequest(http.MethodPost, "/{{.db}}/_config", fmt.Sprintf(collectionsDbConfigUpsertScopes, string(twoCollectionConfigJSON)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(1), importVersion)
+
+	// Update with POST, add collection, version should change
+	response = rt.SendAdminRequest(http.MethodPost, "/{{.db}}/_config", fmt.Sprintf(collectionsDbConfigUpsertScopes, string(threeCollectionConfigJSON)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(2), importVersion)
+
+	// Update with PUT, remove collection, version should not change
+	response = rt.SendAdminRequest(http.MethodPut, "/{{.db}}/_config", fmt.Sprintf(collectionsDbConfigRevsLimit, testBucket.GetName(), string(twoCollectionConfigJSON)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(2), importVersion)
+
+	// Update with POST, remove collection, version should not change
+	response = rt.SendAdminRequest(http.MethodPost, "/{{.db}}/_config", fmt.Sprintf(collectionsDbConfigUpsertScopes, string(oneCollectionConfigJSON)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	importVersion = GetImportVersion(t, rt, dbName)
+	require.Equal(t, uint64(2), importVersion)
+
+}
+
+func GetImportVersion(t *testing.T, rt *rest.RestTester, dbName string) uint64 {
+	var databaseConfig rest.DatabaseConfig
+	_, err := rt.ServerContext().BootstrapContext.GetConfig(rt.Context(), rt.CustomTestBucket.GetName(), rt.ServerContext().Config.Bootstrap.ConfigGroupID, dbName, &databaseConfig)
```

Comment:
nit:

```suggestion
	_, err := rt.ServerContext().BootstrapContext.GetConfig(rt.Context(), rt.Bucket().GetName(), rt.ServerContext().Config.Bootstrap.ConfigGroupID, dbName, &databaseConfig)
```

In case we are able to not use a CustomTestBucket in the future.